.prettierignore

```diff
@@ -1,4 +1,2 @@
-.cache
-package.json
-package-lock.json
```

Comment:
We should ignore `package-lock.json` so that contributors don't accidentally commit it. We use yarn.